src/ngraph/frontend/onnx_import/op/reduce.cpp

```diff
@@ -45,17 +45,19 @@ namespace ngraph
                     auto sum_node = std::shared_ptr<ngraph::Node>{reduction::make_ng_reduction_op(
                         node,
                         node.get_ng_inputs().at(0),
-                        std::make_shared<ngraph::opset0::Sum,
+                        std::make_shared<default_opset::ReduceSum,
                                          const std::shared_ptr<ngraph::Node>&,
-                                         const ngraph::AxisSet&>)};
+                                         const std::shared_ptr<ngraph::Node>&,
+                                         bool>)};
 
                     auto const_node = default_opset::Constant::create(
                         sum_node->get_element_type(),
                         sum_node->get_shape(),
                         std::vector<std::size_t>(shape_size(sum_node->get_shape()),
                                                  elem_count_product));
 
-                    return {std::make_shared<ngraph::opset0::Divide>(sum_node, const_node)};
+                    return {std::make_shared<default_opset::Divide>(
+                        sum_node, const_node, ngraph::op::AutoBroadcastSpec())};
```

Comment:
Just a thought: `sum_node` and `const_node` already match in terms of a shape, so I think we could ommit the last param here.